test/built-ins/Temporal/Duration/prototype/with/argument-wrong-type.js

```diff
@@ -1,27 +0,0 @@
-// Copyright (C) 2021 the V8 project authors. All rights reserved.
-// This code is governed by the BSD license found in the LICENSE file.
-
-/*---
-esid: sec-temporal.duration.prototype.with
-description: Throw TypeError if the temporalDurationLike argument is the wrong type
-features: [Temporal]
----*/
-
-let d = new Temporal.Duration(1, 2, 3, 4, 5);
-
-[
-  "string",
-  "P1YT1M",
```

Comment:
It might be worth having this particular value still in the tests — maybe even in its own file, just to make sure that implementations don't allow a string to be passed to `with()` even if it's a legal Duration string.